pkgs/by-name/cl/clipse/package.nix

```
@@ -0,0 +1,28 @@
+{ 
+  lib, 
+  buildGoModule, 
+  fetchFromGitHub 
+}:
+
+buildGoModule {
+  pname = "clipse";
+  version = "0.0.6";
+
+  src = fetchFromGitHub {
+    owner = "savedra1";
+    repo = "clipse";
+    rev = "v0.0.6";
+    hash = "sha256-zCsSfnZbhOYe1ysnGFO6VYZKVYCNZY0DvP2FB3EXFDM=";
+  };
+
+  vendorHash = "sha256-GIUEx4h3xvLySjBAQKajby2cdH8ioHkv8aPskHN0V+w=";
+
+  meta = with lib; {
+    description = "A useful clipboard manager TUI for Unix.";
+    homepage = "https://github.com/savedra1/clipse";
+    license = licenses.gpl3Only;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ "savedra1" ];
```

Please add yourself to [`maintainers/maintainers-list.nix`](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix) in a separate commit on this PR; see [example PR](https://github.com/NixOS/nixpkgs/pull/273272/commits/0feca8a9559ca4c9372e3a7c2f1c77898e077595) and [Commit conventions](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions) :  
> When adding yourself as maintainer in the same pull request, make a separate commit with the message maintainers: add <handle>. Add the commit before those making changes to the package or module. See [Nixpkgs Maintainers](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) for details.

---

The quotes are no longer necessary now that your GitHub name is added to maintainers-list.nix
```suggestion
    maintainers = with maintainers; [ savedra1 ];
```